notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>The above examples showed open and closed sets in one dimension. Similarly, in two dimensions, open sets can be defined as sets which do not contain their boundaries. The image below shows an example of an open set (a) and a closed set (b) in two dimensions.</p><p> </p><p>NIT: I would switch (a) and (b) so that it is consistent with the previous figure: open sets on the left and closed sets on the right.</p><p> </p><p>"For example, an open set manifold could be a sphere that is not defined at the edge of the sphere." --&gt; I am not sure that this is true. </p><p> </p><p>I would say something like:</p><p>Some manifolds are defined by open sets within an ambient set. For example, the inside of the sphere, i.e. the ball without its boundary, is a manifold that is an open set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>done</p>